### ISSUE TITLE
[error] carthage update

### ISSUE THREAD

**Issue by r-zawislak** [SwiftGit2/SwiftGit2#94]
Hi, I'm trying to install SwiftGit2 for OS X, but i got an error

Version of Xcode: 8.0 (8A218a)
Version of system: 10.12.4 (16E195)

I have installed make, libssh, libtool and automake.

> Warning: cmake-3.8.1 already installed
> Warning: libssh2-1.8.0 already installed
> Warning: libtool-2.4.6_1 already installed
> Warning: automake-1.15 already installed

Also i cleared DerivedData folder and nothing.

> aims:testgit rajmund$ carthage update
> *** Fetching SwiftGit2
> *** Fetching Result
> *** Checking out Result at "3.2.1"
> *** Checking out SwiftGit2 at "v0.3"
> *** xcodebuild output can be found in /var/folders/q_/nr7wws8s2fv1n32x7zwfhfvc0000gt/T/carthage-xcodebuild.wOQqE9.log
> *** Building scheme "Result-watchOS" in Result.xcodeproj
> *** Building scheme "Result-iOS" in Result.xcodeproj
> *** Building scheme "Result-Mac" in Result.xcodeproj
> *** Building scheme "Result-tvOS" in Result.xcodeproj
> *** Building scheme "SwiftGit2-OSX" in SwiftGit2.xcworkspace
> Build Failed
> 	Task failed with exit code 65:
> 	/usr/bin/xcrun xcodebuild -workspace /Users/rajmund/Projects/testgit/Carthage/Checkouts/SwiftGit2/SwiftGit2.xcworkspace -scheme SwiftGit2-OSX -configuration Release -derivedDataPath /Users/rajmund/Library/Caches/org.carthage.CarthageKit/DerivedData/SwiftGit2/v0.3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build
> 
> This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/q_/nr7wws8s2fv1n32x7zwfhfvc0000gt/T/carthage-xcodebuild.wOQqE9.log
> 

The logs from /var/folders/q_/nr7wws8s2fv1n32x7zwfhfvc0000gt/T/[carthage-xcodebuild.wOQqE9.txt](https://github.com/SwiftGit2/SwiftGit2/files/1001589/carthage-xcodebuild.wOQqE9.txt)

The script Script-C9CE0DD61E0710C20053205D.sh : 

> #!/bin/sh
> if which swiftlint >/dev/null; then
>     swiftlint
> else
>     echo "warning: SwiftLint is not installed"
> fi
> 

I can't even compile SwiftGit2-OSX. [logs.txt](https://github.com/SwiftGit2/SwiftGit2/files/1001672/osx_buildlogs.txt)

Also tried 

> github "SwiftGit2/SwiftGit2" "master"

But it doesn't work too. [master_logs.txt](https://github.com/SwiftGit2/SwiftGit2/files/1001682/master_logs.txt)

Any ideas?

**Comment by mdiep** [comment]
It makes sense that the release doesn't work based on the log.

But I'm confused about why `master` would fail. I'm not sure what the issue is. 😕

**Comment by r-zawislak** [comment]
Okey, somehow i managed to make it work. 

Installing new version of Xcode did not change anything.

Then I upgraded and cleared brew by

> brew update && brew upgrade && brew cleanup && brew doctor

After this it did not build at all, so I uninstalled automake, libtool, libssh, cmake, swiftlint, libgit2 and installed it all again. Still nothing. Then I installed libgit2-glib and it run without any problems. Maybe it will help someone.
